jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -1164,7 +1173,23 @@ public Method getDeclaredMethod(String name, Class<?>... parameterTypes) throws
  */
 @CallerSensitive
 public Class<?> getDeclaringClass() {
```

Comment:
There is an existing JavaDoc error for this API.
"This will return null if the receiver is **NOT** a member of another class."
The NOT was missing.
Pls fix it.